common-custom-user-data-gradle-plugin/src/main/java/com/gradle/Utils.java

```diff
@@ -39,12 +36,12 @@ static boolean sysPropertyKeyStartingWith(String keyPrefix) {
         return false;
     }
 
-    static String envVariable(String name) {
-        return System.getenv(name);
-    }
-
-    static boolean envVariablePresent(String name) {
-        return isNotEmpty(envVariable(name));
+    static Optional<String> envVariable(String name) {
+        String value = System.getenv(name);
+        if (isNotEmpty(value)) {
```

Comment:
can you use `Optional.ofNullable` here?

Reply:
We want to check for empty strings as well, and `Optional.ofNullable` doesn't do this.